docs_src/pipelinerun.md

```diff
@@ -1,128 +1,78 @@
+The following are the optional parameters that you can provide as input to `docker-run.sh`. Note that these parameters would affect the performance of the pipeline. 
+    
+- `--classification_disabled`: Disables the classification process of image extraction when applying model. By default, the classification is enabled. 
 
-### `--ocr_disabled`
-To disable Optical character recognition when applying model, default is NOT disabling ocr if this is not provided as input to docker-run.sh
+- `--ocr_disabled`: Disables optical character recognition (OCR) when applying model. By default, OCR is enabled. 
 
-### `--ocr`
-To provide Optical character recogntion frame internal value such as `--ocr 5 GPU`, default recognition interval value is 5. Note device equal to CPU is not supported when executing with a discrete GPU.
+- `--ocr`: Provides the OCR frame internal value, such as `--ocr 5 GPU`. The default recognition interval value is 5. Note device equal to CPU is not supported when executing with a discrete GPU.
 
-### `--barcode_disabled`
-To disable barcode detection when applying model, default is NOT disabling barcode detection if this is not provided as input to docker-run.sh
+- `--barcode_disabled`: Disables barcode detection when applying model. By default, barcode detection is enabled.
+    
+- `--realsense_enabled`: Uses the Intel® RealSense™ Camera and provides the 12-digit serial number of the camera as an input to the `docker-run.sh` script.
 
-### `--realsense_enabled`
-TO use realsense camera and to provide realsense camera 12 digit serial number as inputsrc to the docker-run.sh script
+- `--barcode`: Provides barcode detection frame internal value such as `--barcode 5`, default recognition interval value is 5.
 
-### `--barcode`
-To provide barcode detection frame internal value such as `--barcode 5`, default recognition interval value is 5
+- `--color-width`: Realsense camera color related property, to apply realsense camera color width, which will overwrite the default value of realsense gstreamer; if it's not provided, it will use the default value from realsense gstreamer; make sure to look up to your realsense camera's color capability using `rs-enumerate-devices`
 
-### `--color-width`
-Realsense camera color related property, to apply realsense camera color width, which will overwrite the default value of realsense gstreamer; if it's not provided, it will use the default value from realsense gstreamer; make sure to look up to your realsense camera's color capability using `rs-enumerate-devices`
+- `--color-height`: Realsense camera color related property, to apply realsense camera color height, which will overwrite the default value of realsense gstreamer; if it's not provided, it will use the default value from realsense gstreamer; make sure to look up to your realsense camera's color capability using `rs-enumerate-devices`
```

Comment:
The description of all color parameters are not clear. Please check

Reply:
RealSense cameras produce a color and depth frame. You can use the color-height, color-width, and color-framerate to customize the settings of the color frame output from the camera. There is a depth equivalent but we currently don't' support depth frames so they are not listed.

Reply:
Added this explanation for all three color parameters:

Allows you to customize the settings of the color frame output from the Intel® RealSense™ Cameras. This parameter will overwrite the default value of RealSense gstreamer. Use `rs-enumerate-devices` to look up the camera's color capability.

Reply:
That looks much better!